### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by eksperimental**
I would advice not to merge the squash the commits in this PR

### CODE REVIEW

File: lib/elixir/lib/calendar.ex [hunk 1]
```diff
@@ -1439,7 +1439,7 @@ defmodule DateTime do
   Only supports converting date times which are in the ISO calendar,
   attempting to convert date times from other calendars will raise.
 
-  WARNING: the ISO 8601 does not contain the time zone nor its abbreviation,
+  WARNING: the ISO 8601 date time does not contain the time zone nor its abbreviation,
```
Comment (lexmag) [inline lib/elixir/lib/calendar.ex]: Maybe we should say `ISO 8601 string` instead?

Comment (eksperimental) [inline lib/elixir/lib/calendar.ex]: it's up to you guys

Comment (lexmag) [inline lib/elixir/lib/calendar.ex]: Probably "format" reeds better.
I think we should use same wording in the function's summary as well.

Comment (eksperimental) [inline lib/elixir/lib/calendar.ex]: done

**Comment by eksperimental** [comment]
ready for a final review

**Pull request merged**